### ISSUE TITLE
Can I trainning by adding images to classes trained in Pretrained model(yolov3.weights)?

### ISSUE THREAD

**Issue by sexyHongGi** [pjreddie/darknet#1752]
Hello?
For example, can you add car images and train to create weights that can categorize all 80 classes?

./darknet detector train data/obj.data data/yolov3_test.cfg yolov3.conv.81??
./darknet detector train data/obj.data data/yolov3_test.cfg yolov3.weights??

And can I train by adding only new class images and classify 81 classes?
Thank you..

**Comment by dasmehdix** [comment]
Yes, you can.You can download a well-known dataset like COCO and you can add your custom class into it with your labels and images included(also cfg folders and paths have to be tuned).But you have to train from begginning which means that you can't use pre-trained weights.

I'm not pro.When we train our custom object on pre-traiend weights(transfer learning),before we use detector functions we can maybe tune cfg files and use detection on 81 classes.

**Comment by qq1628qq** [comment]
Hello, I am also interested in this algorithm. How do you train your pre-training model? I also want to train my pre-training model.

**Comment by dasmehdix** [comment]
> Hello, I am also interested in this algorithm. How do you train your pre-training model? I also want to train my pre-training model.

Just follow the  [instruction](https://github.com/AlexeyAB/darknet) of AlexeyAB.
